ipy2to3_other.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
looks like it is a different application. it is not better. it is for different version of ipython